src/test/java/com/rultor/agents/AgentsITCase.java

```diff
@@ -0,0 +1,768 @@
+import com.jcabi.github.Gitignores;
+import com.jcabi.github.Hooks;
+import com.jcabi.github.Issue;
+import com.jcabi.github.IssueEvents;
+import com.jcabi.github.IssueLabels;
+import com.jcabi.github.Issues;
+import com.jcabi.github.Labels;
+import com.jcabi.github.Language;
+import com.jcabi.github.Limits;
+import com.jcabi.github.Markdown;
+import com.jcabi.github.Milestones;
+import com.jcabi.github.Notifications;
+import com.jcabi.github.Organizations;
+import com.jcabi.github.Pulls;
+import com.jcabi.github.Releases;
+import com.jcabi.github.Repo;
+import com.jcabi.github.RepoCommits;
+import com.jcabi.github.Repos;
+import com.jcabi.github.Search;
+import com.jcabi.github.Stars;
+import com.jcabi.github.Users;
+import com.jcabi.github.mock.MkBranches;
+import com.jcabi.github.mock.MkGithub;
+import com.jcabi.http.Request;
+import com.jcabi.http.request.FakeRequest;
+import com.jcabi.immutable.ArrayMap;
+import com.jcabi.ssh.SSHD;
+import com.jcabi.xml.XMLDocument;
+import com.rultor.spi.Profile;
+import com.rultor.spi.Talk;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import java.util.ResourceBundle;
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
+import javax.json.JsonValue;
+import org.apache.commons.lang3.SystemUtils;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Assume;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.mockito.Mockito;
+import org.xembly.Directives;
+
+/**
+ * Integration test for ${@link Agents}.
+ *
+ * @author Denys Skalenko (d.skalenko@gmail.com)
+ * @version $Id$
+ * @since 2.0
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 line)
+ */
+@SuppressWarnings("all")
+public final class AgentsITCase {
+
+    /**
+     * Architect login.
+     */
+    private static final String ARCHITECT = "architect";
+
+    /**
+     * Simple user login.
+     */
+    private static final String USER = "user";
+
+    /**
+     * Message bundle.
+     */
+    private static final ResourceBundle PHRASES =
+        ResourceBundle.getBundle("phrases");
+
+    /**
+     * Temp directory.
+     * @checkstyle VisibilityModifierCheck (5 lines)
+     */
+    @Rule
+    public final transient TemporaryFolder temp = new TemporaryFolder();
+
+    /**
+     * Agents can process a merge.
+     * @throws Exception In case of error.
+     */
+    @Test
+    public void processesMerge() throws Exception {
+        final AgentsITCase.MockGithub github =
+            // @checkstyle MultipleStringLiterals (500 line)
+            new AgentsITCase.MockGithub(new MkGithub("rultor"));
+        final Repo repo = github.randomRepo();
+        final Issue issue = repo.issues().get(
+            // @checkstyle MultipleStringLiterals (1 line)
```

Comment:
@dskalenko this suppression is redundant with the above, also it shouldn't be here in the first place.